lib/state_attr.js

```diff
@@ -338,8 +338,20 @@ const state_attrb = {
 		type: 'boolean',
 		role: 'info',
 		unit:  ''
-	}, 
+	},
+	'supported' : {
+		name: 'supported',
+		type: 'boolean',
+		role: 'info',
+		unit:  ''
+	},
+	'enabled' : {
+		name: 'enabled',
+		type: 'boolean',
+		role: 'info',
```

Comment:
role "info" does not exist in iobroker role definition overview : 
https://github.com/ioBroker/ioBroker/blob/master/doc/STATE_ROLES.md

<img width="459" alt="Schermafbeelding 2020-09-29 om 09 28 53" src="https://user-images.githubusercontent.com/7318445/94526703-32e09280-0236-11eb-9d4e-124b625c1c2f.png">

I think it should. be an alarm type ?

<img width="1104" alt="Schermafbeelding 2020-09-29 om 09 29 41" src="https://user-images.githubusercontent.com/7318445/94526758-4f7cca80-0236-11eb-82d9-4d3aec369361.png">

Reply:
and with indicator.alarm for that.
What do you think?
It's a little bit academically, isn't it?